test/redirector-test.js

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const expect = require('chai').expect;
+const sinon = require('sinon');
+
+
+before(async function () {
+    context = new (require('@dollarshaveclub/cloudworker'))(require('fs').readFileSync('workers/redirector.js', 'utf8')).context;
+    global.Request = context.Request
+    global.URL = context.URL
+    global.handleRequest = context.handleRequest;
+});
+
+describe('Redirector Worker', function() {
+
+    it('returns a 200 for requests that have a matching response, but are not within sample rate', async function () {
+        Math.random = sinon.stub().returns(0.6534);
```

Comment:
Let's use a super low value like 0.0001 here and in the other stubs so that these tests don't break when we lower the sample rate.